tests/hazmat/primitives/test_dh.py

```diff
@@ -18,6 +19,11 @@
 from ...utils import load_nist_vectors, load_vectors_from_file
 
 
+skip_no_dhx = pytest.mark.skipif(
+    backend._lib.Cryptography_HAS_EVP_PKEY_DHX == 0,
```

Comment:
This is an internal implementation detail of OpenSSL so we can't have our skip checks work against this. Other backends won't have this available. If we need to do a check like this we want to do it via some mechanism like `dh_params_supported` or a new backend method to determine whether subgroup order is supported.